requirements/build-requirements.txt

```diff
@@ -1,15 +1,15 @@
-alembic==1.0.2 --hash=sha256:99cc931e11dbef6e41e9376f18be62fc90fe4be9c541eac1b30a3455b3d655f3
-arrow==0.12.1 --hash=sha256:fc8c8e0b587d00f38986bc161f4496e000acea033fe2ce25f4f5bffa9ae53a7c
-certifi==2018.10.15 --hash=sha256:173b19dd31ca7faa50d1fcc0eaf30f5e32e8e99e17d8c7fd4cfc8bc8d94e18a6
-chardet==3.0.4 --hash=sha256:f5632e583a4f61f1e16d0cc98127d241fb11c3c6ddfddee159307d4215186837
-idna==2.7 --hash=sha256:491f674364ba3232ed1eb4c1eb7407887f62cef6c300aad7df6e01acd88ffb25
-mako==1.0.7 --hash=sha256:614c22fe1a5b0a3f46f6c5c43ff2e6795e4e784328d559ec9dc49db0f06b3a75
-markupsafe==1.1.1 --hash=sha256:1733a3715d37d075d79b6fe1c2e1055e8cf471fd7da53cdd284de09782bf75d1
-pathlib2==2.3.2 --hash=sha256:460e67b14d0574b0529a0017b1eb05d10d9722681e303fec7077c2a628de60c1
-python-dateutil==2.7.5 --hash=sha256:f6eb9c17acd5a6954e1a5f2f999a41de3e7e25b6bc41baf6344bd053ec25ceeb
-python-editor==1.0.3 --hash=sha256:e47dcec4ea883853b8196fbd425b875d7ec791d4ede2e20cfc70b9a25365c65b
-requests==2.22.0 --hash=sha256:e3711bd465fd013abe428a8dade2938b578f05d2b06b0ae83daa98869c4548e8
-securedrop-sdk==0.2.0 --hash=sha256:dd0231e1b873197373a7e84e3610370634eead601223551153eca7cdd166e7d5
-six==1.11.0 --hash=sha256:aa4ad34049ddff178b533062797fd1db9f0038b7c5c2461a7cde2244300b9f3d
-sqlalchemy==1.3.3 --hash=sha256:bfb4cd0df5802a01acd738a080a19e60ff4700e030d497de273807f9a8bd4a0a
-urllib3==1.25.10 --hash=sha256:32bcd1ec52a4dd17ede2725ad166c789f50d402870947d6824598915b89ecf70
+alembic==1.0.2 --hash=sha256:ad1e93816f37f8445bb12dfe412579d2237818c9bbf34054a06e00d5f430c700
+arrow==0.12.1 --hash=sha256:e2742eb33011f7aff1d5f27501d802680b6627939f20ff2ef56f050c14af59fa
+certifi==2018.10.15 --hash=sha256:2d5538b9d89b3a9cb423b8f08417c657856cab733d560067e864157f71b1c5aa
+chardet==3.0.4 --hash=sha256:e5cf39014befb85add77118fdc946f0a3387df7840235ba5d20fd6e3a672410a
+idna==2.7 --hash=sha256:a967a752dba5492be34f30f8633cee512d2ec923fe3a2e201d87353b540d2955
+mako==1.0.7 --hash=sha256:99d041a616a3655725dffe459916627b75640a7d045ed33f51ce158a168ca3ef
+markupsafe==1.1.1 --hash=sha256:ab68abcb133e25365454d0572e20c096cfebd3c350b51fe2cf8f54f495e0ca9d --hash=sha256:1733a3715d37d075d79b6fe1c2e1055e8cf471fd7da53cdd284de09782bf75d1
```

Comment:
I see that we have two different wheels for this version of markup:

```
$ sha256sum ../securedrop-debian-packaging/localwheels/MarkupSafe-1.1.1**.whl
ab68abcb133e25365454d0572e20c096cfebd3c350b51fe2cf8f54f495e0ca9d  ../securedrop-debian-packaging/localwheels/MarkupSafe-1.1.1-cp37-cp37m-linux_x86_64.whl
1733a3715d37d075d79b6fe1c2e1055e8cf471fd7da53cdd284de09782bf75d1  ../securedrop-debian-packaging/localwheels/MarkupSafe-1.1.1-py3-none-any.whl
```

Why? Should we maintain both, or remove one?

Reply:
I think it makes sense to remove the old version and just maintain the markupsafe 1.1.1 cp37 wheel

Reply:
Re: I don't see it in https://pypi.org/project/MarkupSafe/#files

Should we also link to the same builds here in our other requirements files, e.g. dev-requirements? I think right now we are using what's on pypi only.

Reply:
Circling back on this: the hashes shown here correspond to those over in the `localwheels/` directory in https://github.com/freedomofpress/securedrop-debian-packaging/pull/213 (soon to be merged into main branch). Contrary to my own comments above, I'm going to leave the duplicate hashes for now, since those values will be automatically pruned on the next dependency update.